ts/src/pro/huobi.ts

```diff
@@ -914,6 +914,7 @@ export default class huobi extends huobiRest {
                     'trades': trades,
                     'status': 'closed',
                     'symbol': market['symbol'],
+                    'filled': parsedTrade['amount'],
```

Comment:
>  I think it's possible that have multiple spot trades in single order

@sc0Vu exactly, we can't rely on a single trade because we might get partial fills